src/Rule.php

```diff
@@ -1403,10 +1403,25 @@ public function dropdownActions($options = [])
             }
         }
 
-        $items = [];
+        $items      = [];
+        $group      = [];
+        $groupname  = ''; // Default group name is empty to match previous no group logic
```

Comment:
You can add a label to have a behaviour similar to what is done on criteria dropdown.

```suggestion
        $groupname  = _n('Action', 'Actions', Session::getPluralNumber());
```